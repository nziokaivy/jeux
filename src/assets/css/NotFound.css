* {
    background-color: #f1f1f1;
}

.not-found-column {
    width: 100%;
    height: 100%;
    padding: 0 1px;
    margin: 0 auto; 
}
.not-found-title {
    background-color: #f1f1f1;
    font-weight: 900;
    margin: 30px;
    text-transform: uppercase;

}
.not-found-image img {
      width: 100%;
      background-color: #f1f1f1;
}
.card {
    margin: 0 auto;
    padding: 16px;
    text-align: center;
    background-color: #f1f1f1;
    display: flex;
    height: 100vh;
}
.not-found-text {
    width: 100%;
    margin-left: 220px;
}

.not-found-text h1{
    width: 100%;
    font-size: 40px;
}

.not-found-text p{
    margin-left: 100px;
    width: 50%;
    font-size: 20px;
    text-align: center;
}

.not-found-link {
    text-decoration: none;
    padding: 10px 20px;
    
}

.not-found-link  p{
    margin-left: 150px;
    margin-top: 40px;
    padding: 10px 0;
    background-color: #5DBCD2;
    color: #000;
    width: 30%;
}
  
/* Responsive columns */
@media screen and (max-width: 600px) {
    .column {
      width: 100%;
      display: block;
      margin-bottom: 20px;
    }
}

