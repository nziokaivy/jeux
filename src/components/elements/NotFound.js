import React from 'react';
import { Link } from 'react-router-dom';
import '../../assets/css/NotFound.css';


const NotFound = () => {
    return (
  <div className="not-found-column">
        <h1 className="not-found-title">JEUX</h1>
    <div className="card">
        <div className="not-found-text">
        <h1>Woops! You've moved <br /> so fast and got lost....</h1>
        <p>The page you are looking for doesn't exist.Please check the URL for spellings or capitalizations.
            If you're having trouble, try visit the Jeux homepage 
        </p>
        <Link className="not-found-link" to="/"><p>Go Home</p></Link>
        </div>
        <div className="not-found-image">
        </div>
    </div>
  </div>
    )
}

export default NotFound;
