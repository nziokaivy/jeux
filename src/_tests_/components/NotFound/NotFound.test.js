import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../../components/elements/NotFound';

describe('Not Found component test with Enzyme', () => {
  test('renders without crashing', () => {
    expect(shallow(<NotFound />)).toMatchSnapshot();
  });
});
