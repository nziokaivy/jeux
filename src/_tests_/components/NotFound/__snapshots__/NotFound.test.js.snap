// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found component test with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;
